api/src/main/java/org/pmiops/workbench/firecloud/FireCloudService.java

```diff
@@ -49,6 +49,12 @@
    */
   void addUserToBillingProject(String email, String projectName);
 
+  /**
+   * Removes the specified user from the specified billing project.
+   * Only used for errored billing projects
+   */
+  void removeUserFromBillingProject(String email, String projectName);
```

Comment:
Question: Does this result in orphaned billing accounts? And if so, will we need some way to prune them over time? I can envision this being an admin process. If this is necessary, can you add a story for that so we don't lose track?

Reply:
Yes, it will. FWIW, our service account will be the owner of all of them. We don't really have anything we can currently do though.